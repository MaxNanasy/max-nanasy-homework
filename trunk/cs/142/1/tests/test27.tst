type x = array 100 of void;main () { }