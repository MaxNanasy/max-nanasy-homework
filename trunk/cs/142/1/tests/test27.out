 TYPE  ident  =  ARRAY  number  OF  VOID  ;  MAIN  (  )  {  }  <EOT> 